Sources/Rendering/Core/Glyph3DMapper/index.js

```diff
@@ -143,6 +145,14 @@ function vtkGlyph3DMapper(publicAPI, model) {
           const orientation = [];
           oArray.getTuple(i, orientation);
           switch (model.orientationMode) {
+            case OrientationModes.MATRIX: {
+              const rotQuat = quat.create();
+              quat.fromMat3(rotQuat, orientation);
+              const rotMat4 = mat4.create();
+              mat4.fromQuat(rotMat4, rotQuat);
```

Comment:
We can just copy the mat3 to the mat4 directly without going through quat, which avoids going through unnecessary calculations.

Reply:
addressed